crates/chainspec/src/spec.rs

```diff
@@ -292,6 +278,12 @@ impl ChainSpec {
         self.chain == Chain::optimism_mainnet()
     }
 
+    /// Returns `true` if this chain is private devnet.
+    #[inline]
+    pub fn is_dev(&self) -> bool {
+        self.chain == Chain::dev()
+    }
```

Comment:
I don't want to add this because this can cause problems for example launch mainnet genesis is --dev mode